client/src/components/Slider/Slider.js

```diff
@@ -0,0 +1,83 @@
+import React from "react";
+import ReactDOM from "react-dom";
+import "./Slider.css";
+
+const Slide = ({ property }) => {
+  const { title, imga, imgb, labela, labelb, index } = property;
+  return (
+    <div id={`slide-${index}`} className="slide">
+      <p className="title slide-text">{`${title}`}</p>
+      <div className="slide-container">
+        <div className="image-1">
+          <img src={imga} className="slide-img" />
+          <p className="slide-text">{`${labela}`}</p>
+        </div>
+        <div className="image-2">
+          <img src={imgb} className="slide-img" />
+          <p className="slide-text">{`${labelb}`}</p>
+        </div>
+      </div>
+    </div>
+  );
+};
+
+class Slider extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      properties: this.props.slideData,
+      property: this.props.slideData[0]
+    };
+  }
+
+  prevProperty = () => {
```

Comment:
Amazing job on the infinite slider here!! This is really cool.

Reply:
Thanks!